test/e2e/components/pill.js

```diff
@@ -0,0 +1,16 @@
+const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('selenium-webdriver');
+
+class Pill extends VlElement {
```

Comment:
niet beter SelectedItem noemen hier?

Reply:
Zou ik niet doen want het is wel degelijk een Pill, weliswaar die van Webuniversum.